src/components/facet/facet_group.test.tsx

```diff
@@ -35,6 +35,13 @@ describe('EuiFacetGroup', () => {
       it('is rendered', () => {
         const component = render(<EuiFacetGroup layout="horizontal" />);
 
+        expect(component).toMatchSnapshot();
+      });
+    });
+    describe('gutterSize', () => {
```

Comment:
Please watch your spacing after closing brackets/parentheses. There should always be an empty line afterward.
```suggestion
    });
    
    describe('gutterSize', () => {
```